UnitySDK/Assets/ML-Agents/Examples/SharedAssets/Scripts/ModelOverrider.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using UnityEngine;
+using Barracuda;
+using System.IO;
+namespace MLAgents
```

Comment:
There is some code in the repo (this code included) that is not for our users to use. I wonder if we should have a separate namespace for those.

Reply:
good idea, I didn't realize they were in the same namespace. maybe `MLAgentsExamples`?

Reply:
Renamed to MLAgentsExamples (except RayPerception, because I'm going to delete that soon).